docs/docs/widgets/checkbox.md

```diff
@@ -1,129 +1,133 @@
 ---
-id: checkbox
-title: Checkbox
----
+
+id: checkbox  
+title: Checkbox  
+---  
 # Checkbox
 
 The **Checkbox** component allows users to make a binary choice, such as selecting or deselecting an option.
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Properties
 
 ### Data
 
 | Property       | Description    | Expected Value         |
 |:---------------|:---------------|:-----------------------|
-| Label          | The text to be used as the label for the checkbox.          | String (e.g., `Select payment preference`).                 |
-| Default status | Sets the default status when the app is loaded.                | Toggle the on/off switch or click on `fx` and dynamically set the value. |
+| Label          | The text to be used as the label for the checkbox.          | String (e.g., **Select payment preference**).                 |
```

Comment:
You have to remove backticks from fx only and rest of the backticks should be untouched, please make this change wherever necessary.